### DISCUSSION TITLE
Steps to change "docs" url not working.

### DISCUSSION THREAD

**Discussion by Web3WithMark** [thuliteio/doks#977] [Q&A] [answered]
Currently using `[0.5.0](https://github.com/h-enk/doks)`.

I am trying to chance the url for `docs` to `tutorial-guides`. 

I have checked the previous questions, there are about 5/6 with different solutions that don't seem to be working. When every I change a permalink, or folder name I get tons of errors about missing files and `Rebuild failed: failed to render pages: render of "section" failed`. 

I even just tried creating a new section using the `npm run create -- --kind docs tutorial-guides`. However when I add content to those sections it does not have the same theme style as docs. (sidebar with sections and content)

So my question, in this current version what are the necessary steps on how do I get the `docs` url to be renames to `tutorial-guides`?

**Comment by h-enk** [comment, marked as answer]
This should do it:

- Create a new section, like you did: `npm run create -- --kind docs tutorial-guides`
- Copy and paste the templates in `./layouts/docs/` to `./layouts/tutorial-guides/`
- Optionally, define the permalink structure to your liking, e.g. `tutorial-guides= "/tutorial-guides/:title/"` (see also https://github.com/h-enk/doks/blob/master/config/_default/config.toml#L65)

**Reply by Web3WithMark** [reply]
You posted that the moment I sat at my computer to try and figure this out haha. Thanks man! it worked

**Reply by felixlyd** [reply]
how to copy `./layouts/docs/` now?  I cannot find the folder when using the latest version.

**Reply by h-enk** [reply]
That step is no longer necessary, it works out of the box (because we're using the default templates now)!

Only make sure to add your newly created section — e.g. "guides" —  to `sectionNav ` in `config/_default/params.toml`

```toml
# Doks (@hyas/doks-core)
[doks]
[..]

  # Nav
  sectionNav = ["docs", "guides"] # ["docs"] (default) or list of sections (e.g. ["docs", "guides"])
  [..]
```

**Comment by ntemposd** [comment]
Hey @h-enk, I'm currently using `v0.4.0`. Is there a way of applying the above solution for me? Thank you.

**Reply by h-enk** [reply]
Hey @ntemposd, the solution above also works with Doks v0.4.0 (tested/verified) 😉

**Reply by ntemposd** [reply]
Oops! Thank you, it works indeed. My confusion was caused because I am using the child theme and there wasn't any `./layouts/docs/` path to copy the templates from. However, I used the ones at https://github.com/h-enk/doks/tree/master/layouts/docs, and worked.

**Reply by ntemposd** [reply]
Dah... right after fixing it and deploying it on production a weird flashing happens all over my website. Any clue why this happens? I hope you can look at the attached to understand what I mean. 

[flashing.webm](https://user-images.githubusercontent.com/2666341/226778433-03eb57ec-14e8-46b8-9fe6-9c48fa23434a.webm)

**Reply by h-enk** [reply]
> My confusion was caused because I am using the child theme and there wasn't any ./layouts/docs/ path to copy the templates from. However, I used the ones at https://github.com/h-enk/doks/tree/master/layouts/docs, and worked.

Technically, you should copy the layouts from `./node_modules/@hyas/doks/layouts/docs` — which corresponds with https://github.com/h-enk/doks/tree/v0.4.0/layouts/docs. This `single.html` is slightly different from the one you copied (but I don't expect that to have an effect on your flashing issue).

> Any clue why this happens?

I think it could be something to do with the order of loading the `.scss`-files. You could try copy over `./node_modules/@hyas/doks/assets/scss/app.scss` and switching these ones:

https://github.com/h-enk/doks/blob/v0.4.0/assets/scss/app.scss#L18-L19

Or, placing `@import "common/dark";` at the bottom

**Reply by ntemposd** [reply]
Nope, sadly this won't fix my issue. When I place `@import "common/dark";` at the bottom, it does not even deploy. I'll try a bit more and then revert to keeping all my content at the original `/docs` path. Thank you anyways. Cheers.
